TicTacToe/TicTacToe.cs

```diff
@@ -32,46 +33,110 @@ public static void GetInput()
             int row = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter Column:");
             int column = int.Parse(Console.ReadLine());
+            playerTurn = (playerTurn == "X") ? "O" : "X";
+            PlaceMark(row, column);
+            
         }
 
         public static void PlaceMark(int row, int column)
         {
-        // your code goes here
+            board[row][column] = playerTurn;
         }
 
         public static bool CheckForWin()
```

Comment:
I am surprised that you are exiting your program from inside the checkforWin method.